Drop-In/src/androidTest/java/com/braintreepayments/api/AddCardFragmentUITest.kt

```diff
@@ -0,0 +1,140 @@
+package com.braintreepayments.api
+
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.lifecycle.Lifecycle
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.typeText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.braintreepayments.api.CardNumber.VISA
+import com.braintreepayments.api.dropin.R
+import com.braintreepayments.cardform.utils.CardType
+import junit.framework.TestCase.assertNull
+import org.hamcrest.CoreMatchers.not
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+
+@RunWith(AndroidJUnit4::class)
+class AddCardFragmentUITest {
+
+    private lateinit var countDownLatch: CountDownLatch
+
+    @Before
+    fun beforeEach() {
+        countDownLatch = CountDownLatch(1)
```

Comment:
What is the importance/benefit of the countDownLatch in all the tests?  Is there some cool tie-in with AndroidJUnit?